tests/test_network.py

```diff
@@ -859,6 +860,7 @@ def test_sld_nad():
                                          depth=10)
         n.write_network_area_diagram_svg(test_svg, low_nominal_voltage_bound=10, depth=10)
         n.write_network_area_diagram_svg(test_svg, high_nominal_voltage_bound=50, depth=10)
+        n.write_network_area_diagram_svg(test_svg, nad_parameters=NadParameters(edge_name_displayed=False))
```

Comment:
False is the default value for this parameter. Is it possible to test True so that we are sure that the user's choice is taken into account?